.github/workflows/tests.yml

```diff
@@ -0,0 +1,96 @@
+name: Tests
+
+# NOTE(mhayden): Restricting branches prevents jobs from being doubled since
+# a push to a pull request triggers two events.
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - master
+
+jobs:
+  lint:
+    name: Lint
```

Comment:
Is there a benefit from having it in one file or just a matter of style, because we have the other workflows in her as well?

Reply:
We can make the workflows dependent on each other. There's no need to build RPMs if unit tests or linting fails. ;)